common/file/UtilTest.cpp

```diff
@@ -90,3 +100,54 @@ void UtilTest::testFilenameFromPath() {
   OLA_ASSERT_EQ(string("baz"), FilenameFromPathOrPath("/foo/bar/baz"));
 }
 
+/*
+ * Inline helper function to check a string ending for a match
+ */
+inline static bool endsWith(const std::string& str, const std::string& suffix) {
+  return (str.size() >= suffix.size()) &&
+          (0 == str.compare(str.size()-suffix.size(), suffix.size(), suffix));
+}
+
+/*
+ * Test the FindMatchingFiles function
+ */
+void UtilTest::testFindMatchingFiles() {
+  bool okay = false;
+
+  std::vector<std::string> prefixes;
+  prefixes.push_back("rdm");
+
+  std::vector<std::string> files;
+
+  okay = FindMatchingFiles(std::string(TEST_SRC_DIR) + PATH_SEPARATOR +
+                           std::string("man"),
+                           std::string("rdm_"), &files);
+
+  OLA_ASSERT_TRUE_MSG(okay, "FindMatchingFiles returned false");
+
+  // At the time this test was written, there were 3 files in folder "man"
+  // starting with "rdm_". If this changed, please adapt the number below
+  // Or find something better to match against
+  OLA_ASSERT_EQ_MSG(3, (int)files.size(),
+                    "Not exactly 3 files man/rdm_* returned");
+
+  bool rdm_model_collector_found = 0;
+  bool rdm_responder_test_found = 0;
+  bool rdm_test_server_found = 0;
+
+  for (const auto& value : files) {
+    if (endsWith(value, "rdm_model_collector.py.1"))
+      rdm_model_collector_found = 1;
```

Comment:
Probably redundant, but is it worth adding some tests that `rdm_model_collector_found == 0` before setting it, to ensure we're not returning duplicates, and someone doesn't fix the test in some weird way in future, like by just bumping the expected count?

Reply:
Yes, I guess that would be redundant since it's a locally-scoped variable. But sure, it cannot hurt to assert that

Reply:
I don't think it would be redundant if we returned ["rdm_model_collector.py.1", "rdm_model_collector.py.1",...] for example.

Thanks for fixing anyway.